include/taskolib/Step.h

```diff
@@ -139,12 +143,21 @@ class Step
      *            inappropriate return value for the step type (see above), if a timeout
      *            is encountered, or if termination has been requested explicitly by the
      *            script.
+     *
+     * \see For more information about step setup scripts see at Sequence.
      */
     bool execute(Context& context)
     {
         return execute(context, nullptr, 0);
     }
 
+    /**
+     * Request if a script is executed.
+     *
+     * \return true for executing a script otherwise false.
+     */
+    bool executes_script();
```

Comment:
If we want to make a member function out of this, it should be `const`.
We could also put it next to `bool requires_bool_return_value(Step::Type step_type)` as a free function. I guess that would be more symmetrical.

Reply:
Fixed. I move it as a free function.